cli/src/main.rs

```diff
@@ -121,21 +129,26 @@ async fn run_test(test_args: TestArgs) -> anyhow::Result<i32> {
         repo_head_commit_epoch.clone(),
     )?;
 
-    if junit_paths.is_empty() {
-        return Err(anyhow::anyhow!("No junit paths provided."));
+    if junit_paths.is_empty() && bazel_bep_path.is_none() {
+        return Err(anyhow::anyhow!("No junit or bazel BEP paths provided."));
     }
 
     let api_client = ApiClient::new(String::from(token))?;
 
     let codeowners = CodeOwners::find_file(&repo.repo_root, codeowners_path);
+    let junit_spec = if !junit_paths.is_empty() {
+        JunitSpec::Paths(junit_paths.clone())
+    } else {
+        JunitSpec::BazelBep(bazel_bep_path.clone().unwrap())
+    };
 
     log::info!("running command: {:?}", command);
     let run_result = run_test_command(
         &repo,
         &org_url_slug,
         command.first().unwrap(),
         command.iter().skip(1).collect(),
-        junit_paths,
+        &junit_spec,
```

Comment:
We can't early parse here like we do in other cases, since we haven't run the test yet 🙃